tools/python_api/test/test_parameter.py

```diff
@@ -4,6 +4,10 @@
 
 import pytest
 from type_aliases import ConnDB
+from typing import TYPE_CHECKING
```

Comment:
Can you add a comment about why we need this?

Reply:
The reason python-lint gave me:
```py
TCH003 Move standard library import `pathlib.Path` into a type-checking block
```